soroban-env-host/src/cost_runner/cost_types/guard_frame.rs

```diff
@@ -12,10 +14,15 @@ impl CostRunner for GuardFrameRun {
     type RecycledType = Option<Self::SampleType>;
 
     fn run_iter(host: &crate::Host, _iter: u64, sample: Self::SampleType) -> Self::RecycledType {
+        let empty_instance = ScContractInstance {
+            executable: ContractExecutable::Token,
+            storage: None,
+        };
```

Comment:
You don't have to anything now, just wanted to mention it. 
The right way to do it is to pass the `empty_instance` from the `sample`, and declare the additional field in the `SampleType`.
The sample set up is done in `benches/common/cost_types/guard_frame.rs` (same name different directory). 
This way we minimize the noise of the setup.

Reply:
Updated.

Reply:
Great, thanks!